src/System.Net.Http/src/System/Net/Http/Managed/AutoRedirectHandler.cs

```diff
@@ -82,7 +82,7 @@ public AutoRedirectHandler(int maxAutomaticRedirections, HttpMessageHandler inne
                 redirectCount++;
                 if (redirectCount > _maxAutomaticRedirections)
                 {
-                    throw new HttpRequestException("max redirects exceeded");
+                    throw new HttpRequestException(SR.net_http_max_redirects);
```

Comment:
Thanks, @davidsh.  Is this the test you're referring to?
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs#L674
It looks like it's verifying that an exception is thrown:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs#L708
It also special cases desktop, with a comment saying that desktop is different and returns a 3xx response rather than throwing:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs#L682
Am I misunderstanding the current situation?